subsys/debug/ppi_trace/ppi_trace.c

```diff
@@ -46,6 +46,13 @@ LOG_MODULE_REGISTER(ppi_trace, CONFIG_PPI_TRACE_LOG_LEVEL);
 #define GET_START_CH(handle) GET_CH(handle)
 #define GET_STOP_CH(handle) (((uint32_t)handle >> 8) & 0xFF)
 
+#if DT_HAS_COMPAT_STATUS_OKAY(nordic_nrf_gpiote)
+#define GPIOTE_IDX DT_PROP(DT_INST(0, nordic_nrf_gpiote), instance)
+static const nrfx_gpiote_t gpiote = NRFX_GPIOTE_INSTANCE(GPIOTE_IDX);
```

Comment:
This assumes that all the pins used in tracing can be handled by a single GPIOTE instance (it would be good to document such limitation somewhere) and that it will be the first one from those enabled in devicetree. I don't think that's safe.